### ISSUE TITLE
close-all

### ISSUE THREAD

**Issue by moi90** [Templarian/MaterialDesign#3708]
It would be great to have a **close-all icon**, similar to check-all.

I would be two crosses overlapping each other.

Maybe this is what the request in #726 was about.

**Comment by goyney** [comment]
I really don't think this works.
![image](https://user-images.githubusercontent.com/1015040/48710782-71877980-ebd7-11e8-9f1b-2914ec8be817.png)

**Comment by moi90** [comment]
@goyney Yes, you are right, this doesn't look right. Maybe if the horizontal shift is increased?

I would want the icon to represent the concept that multiple objects will be rejected/closed. For me, a `close-box-multiple` would not fit 100%.

**Comment by Templarian** [comment]
Added. 😃  Leaving open as there might be a way to represent `close-all`? Not sure though.

**Comment by goyney** [comment]
I do not see another way to represent `close-all` other than what @MrGrigri provided.
